src/cmd/form.go

```diff
@@ -16,16 +17,29 @@ var formCmd = &cobra.Command{
 	You will need to provide the form ID as an argument.
 	The command can be used to retrieve form responses incase you want to pipe it to another command or for testing purposes.
 	`,
-	Args: cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		//TODO: Change to something more meaningful
 		fmt.Println("form called")
```

Comment:
```suggestion
		fmt.Println("Form is being processed with the following arguments:", args)
```

Reply:
I agree with the change in message but throwing in the args wouldn't work in cases where the config file is being used instead

Reply:
I have edited the message